src/server/lib/middleware/login.ts

```diff
@@ -47,14 +61,7 @@ export const loginMiddleware = async (
       next();
     } else {
       trackMetric(Metrics.LOGIN_MIDDLEWARE_NOT_RECENT);
-      if (auth.redirect) {
```

Comment:
Is there a case here where the email is validated, there is a redirect URL, but the status isn't `SIGNED_OUT`? 

And if so, is it okay to fall into the `LOGIN_MIDDLEWARE_NOT_RECENT` case?

Reply:
There are only [3 statuses](https://github.com/guardian/gateway/blob/21763dbea01af34b1a6cdf5ceece32f716fc1c16/src/shared/model/IDAPIAuth.ts#L1):
- Signed Out
- Recently Signed In
- Not Recently Signed In

So all 3 cases have been covered.